tests/test_collection.py

```diff
@@ -115,8 +115,8 @@ def test_name(self):
     def test_mode(self):
         assert self.c.mode == 'r'
 
-    def test_collection(self):
-        assert self.c.encoding == 'iso-8859-1'
+    def test_encoding(self):
+        assert self.c.encoding is None
```

Comment:
GDAL doesn't let us know what is the native encoding of a dataset, so the value we returned before wasn't always accurate. It's more like Python's file object to return None.